ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2

```diff
@@ -89,6 +89,9 @@ daemon TerminAttr
 {%     endif %}
 {%     if daemon_terminattr.sflowaddr is arista.avd.defined %}
 {%         set cvp_config.cli = cvp_config.cli ~ " -sflowaddr=" ~ daemon_terminattr.sflowaddr %}
+{%     endif %}
```

Comment:
I am not sure, but I think these options were put in some order. @noredistribution should we keep these options in a certain order?

Reply:
no, the order doesn't matter
testing this now as well

Reply:
if possible, we like to match the order of what the CLI will return when doing a `show running-config`

Reply:
yea that's true in general, but daemon configs are not ordered, they are as is as they are basically bash statements